src/components/cryptoItem/CryptoItem.tsx

```diff
@@ -0,0 +1,52 @@
+import {View} from 'react-native';
+import {
+  NamesContainer,
+  Contener,
+  Name,
+  Symbol,
+  Logo,
+  ImgeValue,
+  Separador,
+  Value,
+  Percent,
+  ValueContener,
+  ArrowCont,
+} from './cryptoStyle';
+import React from 'react';
+
+const decreaseArrow = require('../../assets/decreaseArrow.png');
+const IncrementArrow = require('../../assets/increaseArrow.png');
```

Comment:
Don't use require, use import like this import IncrementArrow from '../../assets/increaseArrow.png' and paste that name in the source prop